Documentation/ContentBlocks/Carousel.rst

```diff
@@ -0,0 +1,182 @@
+:navigation-title: Carousel
+..  include:: /Includes.rst.txt
+
+..  _carousel:
+
+==============================
+Carousel Content Block example
+==============================
+
+This content block demonstrates some additional features of
+:composer:`friendsoftypo3/content-blocks`. We assume that you are already
+familiar with the concepts described in chapter
+`Custom Content Blocks <https://docs.typo3.org/permalink/t3sitepackage:content-blocks>`_.
+
+..  contents:: Table of contents
+
+..  _carousel-directory:
+
+Directory structure of the carousel content element
+===================================================
+
+Additionally to the files that the jumbotron (compare
+:ref:`content-blocks-jumbotron-directory`) the carousel comes with special CSS
+and JavaScript needed for this element only.
+
+Additionally it supplies a template for its display in the backend.
```

Comment:
I really have no clue about content blocks. Are these directories convention? In extbase context all directories are upper case. So `Assets` and `Templates`. But here all directories are lowercase. Would it be a problem to write these directories upper case? If you have information about it, I would appreciate to add that information here.

Reply:
Yes, they are conventions and have to be lower case. But this should be clear by now, as it's already covered in the basics.

Reply:
As @nhovratov says..